File: lab05/echon.pl
#!/usr/bin/perl
$num_args = @ARGV;
if ($num_args != 2) {
	print "Usage: ./echon.pl num str\n";
	exit;
}
for ($i = 0; $i < $ARGV[0]; $i++) {
	print $ARGV[1];
	print "\n";
}

File: lab06/echon.py
#!/usr/bin/python2.7

import sys

if len(sys.argv) != 3:
	print "Usage: " + sys.argv[0] + " num str"
else:
	i = 0
	while (i < int(sys.argv[1])):
		print sys.argv[2]
		i = i + 1